packages/autocomplete/src/component/Autocomplete.jsx

```diff
@@ -69,7 +75,7 @@ class Autocomplete extends Component<Props, IState> {
   destroy$ = new Subject();
 
   componentDidMount() {
-    const change$ = fromEvent(this.inputField, 'keyup').pipe(
+    const change$ = fromEvent(this.inputField, 'keydown').pipe(
```

Comment:
'keypress'?

Reply:
`keydown` will overwrite the default behaviour when pressing "Enter" in a form (aka: using keydown instead of keyup and keypress will prevent the form from submitting on enter)

Reply:
The cursor is used to highlight the selected option when using keyboard navigation